starters/solidstart-tanstackquery-tailwind-modules/README.md

```diff
@@ -84,6 +84,5 @@ git clone https://github.com/thisdot/starter.dev.git
 
 [Repository](https://github.com/thisdot/starter.dev-github-showcases/tree/main/solidstart-tanstackquery-tailwind-modules)
 
-[Live demo](https://solidstart-tanstackquery-tailwind-modules.starter.dev/)
```

Comment:
instead of deleting completely, you can comment it out 
`<!--[Live demo](https://solidstart-tanstackquery-tailwind-modules.starter.dev/) -->`

Reply:
agree with @hdJerry 👯

Reply:
ok sirs!